components/EntireScheme.svelte

```diff
@@ -68,17 +72,28 @@
   }
 
   function loadFile(e) {
-    const reader = new FileReader();
-    // TODO No await? :(
-    // TODO Should we prompt before deleting the current scheme?
-    reader.onload = (e) => {
-      try {
-        gjScheme.set(backfill(JSON.parse(e.target.result)));
-      } catch (err) {
-        window.alert(`Couldn't load scheme from a file: ${err}`);
-      }
-    };
-    reader.readAsText(e.target.files[0]);
+    otherFiles = [];
+    browseFileIdx = 0;
+
+    let first = true;
+    for (let file of e.target.files) {
+      let immutableFirst = first;
```

Comment:
JS can be a tricky language. :\  In Rust, referring to `first` inside the closure we create would move it there. But in JS, it keeps referring to the outer copy, which we change throughout the loop. Hence this copy